bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -276,10 +276,10 @@ jobject process_completed_stream_recv(size_t length, jobject callback)
 {
     JNIEnv *env = get_jni_env();
     jobject jucx_request = env->NewObject(jucx_request_cls, jucx_request_constructor, NULL);
+    env->SetLongField(jucx_request, recv_size_field, length);
```

Comment:
can we reuse set_jucx_request_completed?

Reply:
This is a path where we don't have request. We can but would need to allocate temp request, set length there, that would be more lines.

Reply:
i see, do we need to set native id for jucx_request here?

Reply:
We set `jobject jucx_request = env->NewObject(jucx_request_cls, jucx_request_constructor, NULL)`
Last argument sets nativeId to null,